docs/gaming/nfts-in-games.md

```diff
@@ -0,0 +1,160 @@
+---
+title: NFTs in games
+head:
+  - - meta
+    - name: title
+      content: Solana Cookbook | NFTs in games
+  - - meta
+    - name: og:title
+      content: Solana Cookbook | NFTs in games
+  - - meta
+    - name: description
+      content: A list of options that should give you an idea of how NFTs can be utilized in games
+  - - meta
+    - name: og:description
+      content: A list of options that should give you an idea of how NFTs can be utilized in games
+  - - meta
+    - name: og:image
+      content: https://solanacookbook.com/cookbook-sharing-card.png
+  - - meta
+    - name: og:image:alt
+      content: Solana splash card
+  - - meta
+    - name: twitter:card
+      content: summary
+  - - meta
+    - name: twitter:site
+      content: "@solanacookbook"
+  - - meta
+    - name: twitter:image
+      content: "https://solanacookbook.com/cookbook-sharing-card.png"
+  - - meta
+    - name: robots
+      content: index,follow,noodp
+  - - meta
+    - name: googlebot
+      content: index,follow
+footer: MIT Licensed
+---
+
+# NFTs In Games
+
+
+NFTs are probably the easiest and most popular way to integrate Solana into your game.
+NFTs are objects that are saved on the Solana block chain and have a json metadata attached to them which can be used to safe attributes about the NFT. These are perfect for usage in games.
+
+## Allow only players from a collection
+
+- You can for example only allow players that own an NFT of an NFT collection to enter the game.
```

Comment:
```suggestion
Using NFTs, you can gate access to a particular part of a game based on owning the NFT. This can form a more tightly-knit community within your game.
```